heliopy/data/util.py

```diff
@@ -737,12 +738,24 @@ def cdf2df(cdf, index_key, dtimeindex=True, badvalues=None, ignore=None):
     ignore : list, optional
         In case a CDF file has columns that are unused / not required, then
         the column names can be passed as a list into the function.
+    include : str, list, optional
```

Comment:
This looks like a neat addition that could standalone - would it be possible to put the changes in this file to another PR that just adds this keyword and the associated logic?

Reply:
Done. See pull request [#841](https://github.com/heliopython/heliopy/pull/841).